internal/devconfig/configfile/packages.go

```diff
@@ -366,3 +368,134 @@ func packagesFromLegacyList(packages []string) []Package {
 	}
 	return packagesList
 }
+
+// PackageSpec specifies a Devbox package to install. Devbox supports a number
+// of package spec syntaxes:
+//
+//	| Syntax           | Example                     | Description                          |
+//	| ---------------- | --------------------------- | ------------------------------------ |
+//	| name@version     | go@v1.22                    | Resolved w/ search service           |
+//	| name             | go                          | Same as above; implies @latest       |
+//	| flake            | github:/nixos/nixpkgs#go    | Matches Nix installable syntax       |
+//	| attr path        | go                          | Equivalent to flake:nixpkgs#go       |
+//	| legacy attr path | go                          | Uses deprecated nixpkgs.commit field |
```

Comment:
Is it deprecated? We removed it because we don't really want anyone to use it. I guess our preference would be for folks to use the hash in the package they want?

Reply:
Yeah, our docs say it's deprecated and will eventually be removed. I think pinning using a revision in a flake reference is clearer.